cpp/src/neighbors/cagra_search_half.cu

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2023-2024, NVIDIA CORPORATION.
```

Comment:
Here and in other files newly moved from raft, shall we have the copyright starting year copied from raft, or reset the counter to 2024?

Reply:
If an earlier start year exists, by copying from RAFT, don’t reset it. The copyright script only enforces that the end year is current. Leave start years at the earlier of (whatever git log says) and (a year in which we know the code existed in RAFT or other NVIDIA libraries).

Reply:
Thanks @bdice for the clarification!